javascript/operators/await.json

```
@@ -62,6 +62,75 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "top_level": {
+          "__compat": {
+            "description": "Use at module top level",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Operators/await",
+            "spec_url": "https://tc39.es/proposal-top-level-await/#introduction",
+            "support": {
+              "chrome": {
+                "version_added": false,
+                "notes": "See <a href='https://crbug.com/9344'>bug 9344</a>"
```

This link doesn't look right. I'm guessing a digit is missing. Could you find the correct link and fix it?

---

Oops, https://bugs.chromium.org/p/v8/issues/detail?id=9344 is the bug I tried linking to. Should I just use the full URL?

---

I got it, this should work
```suggestion
                "notes": "See <a href='https://crbug.com/v8/9344'>bug 9344</a>"
```